images-list

```diff
@@ -115,12 +115,16 @@ gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provide
 gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provider-vsphere-cpi-release-manager v1.21.3
 gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provider-vsphere-cpi-release-manager v1.22.5
 gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provider-vsphere-cpi-release-manager v1.22.6
+gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provider-vsphere-cpi-release-manager v1.22.7
 gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provider-vsphere-cpi-release-manager v1.23.0
 gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provider-vsphere-cpi-release-manager v1.23.0-alpha.1
 gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provider-vsphere-cpi-release-manager v1.23.1
+gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provider-vsphere-cpi-release-manager v1.23.2
```

Comment:
Per issue description, we need the latest v1.23.3 - do we also need v1.23.2?

Reply:
I'm not aware why we would have to have v1.23.2 if we don't plan on referencing it. @annablender , do you know why we might need to have it?

Reply:
Correct, I figured yes since it may be used for the constraints.. but perhaps not? I think I'm bumping straight to 1.23.3 here https://github.com/rancher/vsphere-charts/blob/c433c58733e91515157454e46310f8cb58732bea/charts/rancher-vsphere-cpi/values.yaml#L44 so 1.23.2 is not needed. @jiaqiluo Any thoughts?

Reply:
@annablender , constraints are for k8s versions. Since minor versions of CPI charts are aligned with minor versions of k8s they are for, it may be a bit confusing. But constraints are keyed off of k8s versions. :)

Reply:
Yep, we should mirror 1.23.3 and skip 1.23.2